apps/editing-toolkit/editing-toolkit-plugin/block-patterns/class-block-patterns-from-api.php

```diff
@@ -236,5 +241,51 @@ private function can_register_pattern( $pattern ) {
 
 		return true;
 	}
+
+	/**
+	 * Unregister all core patterns, then reregister core patterns in core Wordpress only,
+	 * that is those in wp-includes/block-patterns.php
+	 * Gutenberg adds new and overrides existing core patterns. We don't want these for now.
+	 */
+	private function reregister_core_patterns() {
+		if ( class_exists( 'WP_Block_Patterns_Registry' ) ) {
+			foreach ( \WP_Block_Patterns_Registry::get_instance()->get_all_registered() as $pattern ) {
+				// Gutenberg registers patterns with varying prefixes, but categorizes them using `core/*` in a blockTypes array.
```

Comment:
Thanks for flagging this.

>  I can't quite think of another way to target those Gutenberg patterns.

It seemed too unruly to add exceptions for `query/*` and other Gutenberg pattern names, especially if WPCOM wishes to add its own too.

> should we add a "todo" item somewhere to revisit it

You bet. Will do.

> Or propose in Gutenberg to flag the experimental ones in some way so that they can be more easily excluded

Both I think sound über reasonable. Great idea. 🙇

Reply:
I've found a [patterns tracking issue.](https://github.com/WordPress/gutenberg/issues/31153) I've asked there whether it's worth creating an issue.

Reply:
Sounds good, thanks for that!